### ISSUE TITLE
Use case for Audio

### ISSUE THREAD

**Issue by RandomMetalhead** [nicoduj/homebridge-harmony#181]
Hi together,

First of all, thanks a lot for the effort with this plugin!

I am having a use case, which is not fitting perfectly and maybe there is a ootb solution, I am not seeing at the moment. 

Following setup:
- Plugin in TV Plattform Mode
- Philips TV
- Audio System controlled via HarmonyHub

Problem:
When I start my audio system, TV in HomeKit also starts, because “Audio System” is a Input for TV. I also use automation to dimm the lights, when I watch TV. I don’t want to dimm the lights when I am listening music. The automation is triggered by TV. So even when I start only the audio system, my light gets dimmed, because the TV turns on with input “audio”. 

Is there a easy solution for this problem?

**Comment by nicoduj** [comment]
Hi,

What is your audio system ? You can always exclude the avtivity from tv and add a switch for it for example i think (options activitiesToPublishAsAccessoriesSwitch And activitiesToPublishAsInputForTVMode)

**Comment by RandomMetalhead** [comment]
Hi,

I tried to show my usecase in a image.

<img width="1792" alt="Bildschirmfoto 2019-08-27 um 21 00 07" src="https://user-images.githubusercontent.com/1662570/63800269-1032ca80-c90e-11e9-9ecc-b97ba18c73a9.png">

config looks like this

```
{
            "platform": "HarmonyHubWebSocket",
            "name": "HarmonyHub",
            "hubIP": "10.10.1.29",
            "activitiesToPublishAsInputForTVMode": [
                "Apple TV schauen",
                "Playstation spielen"
            ],
            "activitiesToPublishAsAccessoriesSwitch": [
                "Anlage aktivieren"
            ]
        }
```

Problem: I use the "TV turned on" for automatically turn down lights via automation. Which I do not want with Audio. So TV should not turn on, when using switch for Audio (Anlage aktivieren).

In the meanwhile, your plugin has become very powerful and option rich. Maybe I missed a usecase for another option.

Thanks!

**Comment by nicoduj** [comment]
Understood, i think i know what is happening will try to fix (Seems same thing that #178)

**Comment by RandomMetalhead** [comment]
Works perfectly. Thanks for your hard work!
